doc/cla/individual/flotchet.md

```diff
@@ -0,0 +1,11 @@
+Belgium, 08/08/2023
+
+I hereby agree to the terms of the Odoo Individual Contributor License
+Agreement v1.0.
+
+I declare that I am authorized and able to make this agreement and sign this
+declaration.
+
+Signed,
+
+Florent Maisse. flo.maisse@live.be https://github.com/Flotchet
```

Comment:
Not sure this should be in this PR

Reply:
No it should not. @Flotchet use your @odoo.com email address for your commits